mars/services/task/supervisor/processor.py

```diff
@@ -599,11 +599,27 @@ def get_tileable_details(self):
             else:
                 status = SubtaskStatus.running
 
+            # get properties for tileable
+            all_propertes = {'slot': getattr(tileable, slot, None) for slot in tileable.__slots__}
```

Comment:
I wonder if this list comprehension expression will work. Please also try adding tests for your code.